### ISSUE TITLE
Virus warning due to unsigned exe file

### ISSUE THREAD

**Issue by ganeshh123** [ganeshh123/gta5-cleaner#8]
Downloading the file displays a warning in the browser and may be blocked by Windows:
![image](https://user-images.githubusercontent.com/43777439/182603357-d22614e4-49ff-44c7-85b4-2a8ba69a4dbc.png)
https://support.google.com/chrome/answer/2898334?visit_id=637951250447563243-1473003491&hl=en&rd=2#virus&zippy=%2Cvirus-scan-failed-or-virus-detected

The file needs to be signed for this to stop.

**Comment by ganeshh123** [comment]
nexe/nexe#945

**Comment by ganeshh123** [comment]
Related to nexe/nexe#945

**Comment by Michael21107** [comment]
@ganeshh123 appeared also (or reappeared) in [1.2.4](https://github.com/ganeshh123/gta5-cleaner/tree/1.2.4)
![image](https://github.com/user-attachments/assets/74018df8-20c2-4622-9edb-31281efaf449)

**Comment by ganeshh123** [comment]
Yeah, I can't do anything. Not willing to pay thousands of dollars for a certificate.

**Comment by Michael21107** [comment]
can't you ''sign it'' or something for cheaper?
